packages/create-cafs-store/src/index.ts

```diff
@@ -80,6 +80,7 @@ export default function createCafsStore (
   })
   return {
     ...createCafs(cafsDir, opts?.ignoreFile),
+    cafsDir,
```

Comment:
do you still need this? Now getFilePathInCafs is exported.

Reply:
Yes, I still need it, because I need to know where to link the package to.